django_server/.github/workflows/deploy.yml

```
@@ -0,0 +1,218 @@
+name: Deploy
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)